File: Algoritimos-fundamentais-Entregavel1/Fibonacci.js
const readline = require("readline");

const teclado = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

teclado.question("Digite a quantidade de termos: ", function(n) {

    n = parseInt(n);

    let a = 0;
    let b = 1;
    let proximo;

    for (let i = 0; i < n; i++) {
        console.log(a);

        proximo = a + b;
        a = b;
        b = proximo;
    }

    teclado.close();
});

File: Algoritimos-fundamentais-Entregavel1/Fibonacci.java
import java.util.Scanner;

public class Fibonacci {

    public static int calcularProximo(int a, int b) {
        return a + b;
    }

    public static void main(String[] args) {

        Scanner teclado = new Scanner(System.in);

        int n;
        int a = 0;
        int b = 1;
        int proximo;

        System.out.print("Digite a quantidade de termos: ");
        n = teclado.nextInt();

        for (int i = 0; i < n; i++) {
            System.out.print(a + " ");

            proximo = calcularProximo(a, b);
            a = b;
            b = proximo;
        }

        teclado.close();
    }
}